src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py

```diff
@@ -226,17 +224,26 @@ def acr_update_set(client,
     return client.update(resource_group_name, registry_name, parameters)
 
 
-def acr_login(registry_name, resource_group_name=None, username=None, password=None):
+def acr_login(registry_name,  # pylint: disable=too-many-statements
```

Comment:
Instead of adding this pylint disable, could you please try to break this function into smaller units?